app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
Apparently it has a `#:nodoc:` on it: https://github.com/rails/rails/blob/d5902c9e7eaba4db4e79c464d623a7d7e6e2d0e3/activerecord/lib/active_record/sanitization.rb#L11